src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor().name();
+        Ball ball = new Ball(randomColor, randomNumber);
+        return ball;
```

Comment:
```suggestion
        return new Ball(randomColor, randomNumber);
```
![image](https://user-images.githubusercontent.com/81781479/189151267-c92b77d9-0be3-4863-a14d-87df17540535.png)